macrosynergy/management/check_availability.py

```diff
@@ -99,7 +103,12 @@ def visual_paneldates(df: pd.DataFrame, size: Tuple[float] = None):
         df = df.apply(pd.to_datetime)
         # All series, in principle, should be populated to the last active release date
         # in the DataFrame.
-        maxdate = df.max().max()
+        
+        if use_last_businessday:
+            maxdate: pd.Timestamp = pd.Timestamp.today() - pd.tseries.offsets.BusinessDay()
```

Comment:
Need to make it a date:
```python
maxdate: datetime.date = (pd.Timestamp.today() - pd.tseries.offsets.BusinessDay()).date()
```
As real date doesn't have a time dimension to it.

Reply:
The underlying function `business_day_dif()` that calculates the differences needs a `pd.Timestamp`